wa/workloads/androbench/__init__.py

```diff
@@ -0,0 +1,54 @@
+#    Copyright 2014-2016 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import re
+
+from wa import ApkUiautoWorkload, Parameter
+
+class Androbench(ApkUiautoWorkload):
+
+    name = 'androbench'
+    package_names = ['com.andromeda.androbench2']
+    regex_matches = [re.compile('Sequential Read Score ([\d.]+)'), re.compile('Sequential Write Score ([\d.]+)'),
+    re.compile('Random Read Score ([\d.]+)'), re.compile('Random Write Score ([\d.]+)'), re.compile('SQL Insert Score ([\d.]+)'),
+    re.compile('SQL Update Score ([\d.]+)'), re.compile('SQL Delete Score ([\d.]+)')]
```

Comment:
Could you split this up with a single regex per line to make this easier to read?
e.g.
```
regex_matches = [re.compile('Sequential Read Score ([\d.]+)'), 
                 re.compile('Sequential Write Score ([\d.]+)'),
                 re.compile('Random Read Score ([\d.]+)'),
                 re.compile('Random Write Score ([\d.]+)'), 
                 re.compile('SQL Insert Score ([\d.]+)'),
                 re.compile('SQL Update Score ([\d.]+)'), 
                 re.compile('SQL Delete Score ([\d.]+)')]